charts/dependabot-gitlab/values.yaml

```diff
@@ -113,6 +113,22 @@ worker:
     # -- Startup probe timeout
     timeoutSeconds: 3
 
+  serviceMonitor:
+    enabled: true
+    additionalLabels: {}
+    # The label to use to retrieve the job name from.
+    # jobLabel: "app.kubernetes.io/name"
+    namespace: ""
+    namespaceSelector: {}
```

Comment:
As I mentioned in previous comment, I think it's better not to have this configurable. If this serviceMonitor is deployed by this chart, it should scrape only objects created by this chart.